Zend/zend_language_parser.y

```diff
@@ -601,6 +601,18 @@ class_modifiers:
 			{ $$ = zend_add_class_modifier($1, $2); if (!$$) { YYERROR; } }
 ;
 
+anonymous_class_modifiers:
+		class_modifier
+			{ $$ = zend_add_anonymous_class_modifier(0, $1); if (!$$) { YYERROR; } }
+	|	anonymous_class_modifiers class_modifier
+			{ $$ = zend_add_anonymous_class_modifier($1, $2); if (!$$) { YYERROR; } }
+;
+
+anonymous_class_modifiers_optional:
+		%empty			{ $$ = 0; }
+	|	anonymous_class_modifiers
```

Comment:
`{ $$ = $1; }` for consistency?

Reply:
Sure. Amended the commit, thanks for reviewing.